sig-api/charter.md

```diff
@@ -0,0 +1,22 @@
+# SIG API
+
+Weekly meetings on Wednesdays at 13:00 UTC.
+https://docs.google.com/document/d/1LCLt5k9x1yISZE_0SxGb5hcU3Q0pzDqRfIMEffgqlnk/edit?usp=sharing
+
+## Scope
+* Define and drive graceful, forward and backward compatible API changes in within KubeVirt
```

Comment:
You do not want to "Define … API changes".

Can we reword this? Maybe you want to say that you want to _support the community_ in doing "graceful, fwd and etc changes?"